### ISSUE TITLE
JOSS Review - Specific 6, 7, 9 - R programming usage

### ISSUE THREAD

**Issue by PsyChiLin** [PsyChiLin/EFAshiny#7]
Specific point 6, 7, 9 from [JOSS Review](https://github.com/openjournals/joss-reviews/issues/567):

6. It would be great to be able to extract figures in R (in ggplot format or so), to further use them in rmarkdown reports or customize them.

7. In the same vein, a button to get the R code used to generate the results would be very useful for learning R and implement automated processing scripts.

9. In general, the feature that lacks the most is the easy possibility to interface your app with a programming R usage. In other words, the possibility to use your package within a more general script pipeline. That includes the possibility to extract the data (for example, the loadings with their confidence intervals), the code used at each step to generate the results etc., to be able to further reproduce the entire analysis without launching the app and clicking on the buttons.

**Comment by PsyChiLin** [comment]
Thanks for these comments. I have added a `Editor` tab with several quick examples to allow users to learn how to perform analyses or produce plots similar to those in `EFAshiny` GUI. I think a button to get R code is not necessary, because users can use the `showcase` function in `shiny` to understand the underlying code. Instead, they need some useful and easy examples to learn R and further automate their analyses in the near future. Thus, I have designed a `Editor` tab for users who use the Github version of `EFAshiny`. They can adopt `EFAshiny` within a more general script framework as you suggested right now haha.
In summary, users who want to further understand EFAshiny or learn R can (1) see the code in `Editor` tab of `EFAshiny` GUI, (2) download the [R markdown file](https://github.com/PsyChiLin/EFAshiny/blob/master/EFAshiny_RCodeDemo.Rmd) similar to the code in editor mode, (3) see the same R markdown file in [this public link](http://rpubs.com/Chi-Lin/EFAshiny_R_Code_Demo), (4) use `showcase` function in `shiny`, and (5) directly see the code in `server.R` and `ui.R`.
Have also updated information in the paper and the README file.
Thanks for useful suggestions !

**Comment by PsyChiLin** [comment]
As shown in the reply, we have addressed this issues in latest version. All information are provided in the tutorial file for users to implement these tools.
